### PR TITLE
Upgrade github.com/ugorji/go/codec

### LINKED ISSUE #2726: failed to go build -tags="appengine"
env:
go 1.16.4
Windows 10
use example code from Quick start section

```
go build -tags="appengine"
# github.com/ugorji/go/codec
..\..\go\pkg\mod\github.com\ugorji\go\codec@v1.1.7\encode.go:651:9: undefined: mapIter
..\..\go\pkg\mod\github.com\ugorji\go\codec@v1.1.7\encode.go:652:2: undefined: mapRange
```

if upgrade codec to latest version(1.2.6), there is no error will be return.

### PR DESCRIPTION
**Pull request by sunshineplan**
Fixes #2726

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,10 +5,11 @@ go 1.13
 require (
 	github.com/gin-contrib/sse v0.1.0
 	github.com/go-playground/validator/v10 v10.4.1
+	github.com/goccy/go-json v0.5.1
```
Comment (appleboy) [inline go.mod:8]: Why change this?

Comment (sunshineplan) [reply go.mod:8]: It seems to be automatic generated when I run go mod tidy after upgrading github.com/ugorji/go/codec

**Review by thinkerou — DISMISSED** [review]

**Comment by appleboy** [comment]
@sunshineplan Please fix the conflicts.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
@sunshineplan Thanks

**Pull request merged**